src/FloatingButton/FloatingButton.tsx

```diff
@@ -1,16 +1,16 @@
-import * as _ from "lodash";
-import * as classnames from "classnames";
+import "./FloatingButton.less";
+
 import * as PropTypes from "prop-types";
 import * as React from "react";
 import * as RootCloseWrapper from "react-overlays/lib/RootCloseWrapper";
+import * as _ from "lodash";
+import * as classnames from "classnames";
 
 import { Button } from "../Button/Button";
 import FlexBox from "../flex/FlexBox";
 import { Icon } from "../Icon/Icon";
-import { breakpointS } from "../utils/Constants";
 import { Values } from "../utils/types";
-
-import "./FloatingButton.less";
+import { breakpointS } from "../utils/Constants";
```

Comment:
Not a fan of how these were ordered, eh 😛

Reply:
My formatter did this! Hmm is this not all right?

Reply:
Ah I figured it was a formatter thing. Which formatter do you use? It's not a big deal, but I was confused for a second trying to see which dependency was added or removed.

Reply:
I don't have anything special set up, i just have set eslint to format the doc when I save it. I think it would follow the rules set in the formatting files in the repo?